src/attrib.d

```diff
@@ -37,13 +37,13 @@ import ddmd.tokens;
 import ddmd.utf;
 import ddmd.visitor;
 
-/**************************************************************/
+/***********************************************************
```

Comment:
Can't we just delete these?  They were much more useful in C++ where the function bodies were not inside the class declarations.